bioimageio/spec/shared/schema.py

```diff
@@ -113,7 +113,9 @@ def matching_lengths(self, data, **kwargs):
 class ImplicitOutputShape(SharedBioImageIOSchema):
```

Comment:
if we are very diligent we'd add validation such that offset is not zero where scale is `None`...